doc/man/zcashd.1

```diff
@@ -72,9 +72,14 @@ Imports blocks from external blk000??.dat file on startup
 .IP
 Keep at most <n> unconnectable transactions in memory (default: 100)
 .HP
+\fB\-mempooltxinputlimit=\fR<n>
+.IP
+Set the maximum number of transparent inputs in a transaction that the
+mempool will accept (default: 0 = no limit applied)
+.HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-4\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-2\fR to 16, 0 = auto, <0 =
```

Comment:
Is this change just because the man page update script happened to be run on a machine/VM with two cores?